install_config/aggregate_logging.adoc

```diff
@@ -953,6 +953,67 @@ Further explanation of how to set up the `fluent-plugin-secure-forward` plug-in
 can be link:https://github.com/tagomoris/fluent-plugin-secure-forward[found
 here].
 
+[[reduce-numer-of-connections-from-fluentd]]
+*Reduceing the Number of Connections from Fluentd to the API Server*
```

Comment:
Just a suggestion, but we could do a[ discrete heading ](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#discrete-headings )for this section.